### ISSUE TITLE
Rotation info in a NEF file

### ISSUE THREAD

**Issue by photopea** [exiftool/exiftool#2]
Hi, when I process the attached NEF file with exiftool, it detects a `Rotation: 270.` I need to programmatically extract this information form a file, but can not find it anywhere (not in Shot Info, neither in IPTC data, neither in XMP, neither in Exif 0xbc02). Where did Exiftool find it?
[portrait.zip](https://github.com/exiftool/exiftool/files/2715935/portrait.zip)

**Comment by exiftool** [comment]
This comes from the Nikon maker notes.  See here for a complete list of Nikon tags extracted:

http://owl.phy.queensu.ca/~phil/exiftool/TagNames/Nikon.html

**Comment by photopea** [comment]
Thanks for the answer! According to your link, only Shot Info - tag  145 (0x91) -  can contain the information about Rotation / Orientation. I looked into the MakerNote of my file. There is a tag  145 (0x91), which apparently contains "Nikon ShotInfoD3a". 

I looked into the definition of ShotInfoD3a here: 

https://github.com/exiftool/exiftool/blob/bd34970d642d8bd6191909ac857a075be5468275/lib/Image/ExifTool/Nikon.pm 

and apparently, it does not contain any Rotation information :( Is it possible, that ExifTool found the rotation information elsewhere, maybe completely outside of MakerNote?
